components/landing/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { FC } from "react";
+import Image from "next/image";
+
+export interface LandingProps {}
+
+const Landing: FC<LandingProps> = () => {
+  return (
+    <div className="Landing">
+      <div className="Landing__ImageContainer">
+        <img src="/Headshot.jpg" className="Landing__Image" />
+      </div>
+      <div className="Landing__TextContainer">
+        <div className="Landing__Text__Name">Brad Bieselin</div>
+        <div className="Landing__Text__Freelance">-Freelance Web Developer</div>
+        <div className="Landing__Text__Company">-Company: Witbe Inc</div>
+        <div className="Landing__Text__CompanyRole">
+          -Position: Project Manager
+        </div>
+        <div className="Landing__Text__School">
+          -2019 B.S. Computer Science Graduate
+        </div>
+        <div className="Landing__Text__Long">
+          -Experience in professional and academic environments
+        </div>
+      </div>
+      <style jsx>
+        {`
+          .Landing {
+            width: 100%;
+            background: black;
+          }
+          .Landing__Image {
+            padding-top: 7rem;
+            width: 100%;
+            height: auto;
+          }
+          .Landing__Text {
+            flex-wrap: wrap;
+            text-align: center;
+            color: white;
+            padding-bottom: 1rem;
+          }
+          .Landing__Text__Long {
+            flex-wrap: wrap;
+            text-align: left;
+            color: white;
+            padding-bottom: 1rem;
+            font-size: 12px;
+          }
+          .Landing__Text__Name {
+            flex-wrap: wrap;
+            text-align: Landing__Text__Freelance;
+            color: white;
+            padding-bottom: 1rem;
+            font-size: 50px;
+          }
+          .Landing__Text__Freelance {
+            flex-wrap: wrap;
+            text-align: left;
+            color: white;
+            padding-bottom: 1rem;
+            font-size: 25px;
+          }
+          .Landing__Text__Company {
+            flex-wrap: wrap;
+            text-align: left;
+            color: white;
+            padding-bottom: 1rem;
+            font-size: 25px;
+          }
+          .Landing__Text__CompanyRole {
+            flex-wrap: wrap;
+            text-align: left;
+            color: white;
+            padding-bottom: 1rem;
+            font-size: 25px;
+          }
+          .Landing__Text__School {
```

Comment:
Consistency with naming convention here as well

Reply:
Changed naming convention for consistency